functions/webhooks/twitter/FlexToTwitter.protected.ts

```diff
@@ -68,60 +67,57 @@ export const handler = async (
   const resolve = bindResolve(callback)(response);
 
   try {
-    const { recipientId, Body } = event;
-
+    const { recipientId, Body, From, ChannelSid, EventType, Source } = event;
     if (recipientId === undefined) {
```

Comment:
I think the only possible values here are string or undefined, since this function is actually [passed as a callback from our code](https://github.com/techmatters/serverless/blob/4a9df561844ff076758e76591a492db2d56590a2/functions/webhooks/twitter/TwitterToFlex.ts#L168) (and nowhere else it's being used).

Happy to consider adding more advance validation tho, is true that this could be "empty string" and in that case we'll end with a 5xx status code.

Reply:
I was suggesting simpler validation, just `if (!recipientId) {` but it doesn't matter. I only brought it up because it caused some of the old tests to break but those are fixed now.